table/block.h

```diff
@@ -283,16 +284,10 @@ class BlockIter final : public InternalIterator {
   virtual Status status() const override { return status_; }
   virtual Slice key() const override {
     assert(Valid());
-    return key_includes_seq_ ? key_.GetInternalKey() : key_.GetUserKey();
+    return key_.GetInternalKey();
   }
   virtual Slice value() const override {
     assert(Valid());
-    if (read_amp_bitmap_ && current_ < restarts_ &&
-        current_ != last_bitmap_offset_) {
-      read_amp_bitmap_->Mark(current_ /* current entry offset */,
-                             NextEntryOffset() - 1);
-      last_bitmap_offset_ = current_;
-    }
     return value_;
   }
```

Comment:
Can we leave this function not implemented?

Reply:
Meta blocks could still legitimately use BlockIter class.